.github/workflows/test-runner-on-perf.yml

```diff
@@ -8,14 +8,40 @@ env:
   GOOGLE_ZONE: us-central1
   K8_CLUSTER: jade-master-us-central1
 on:
-  schedule:
-    - cron: '0 4 * * *' # run at 4 AM UTC, 12PM EST.
+  #schedule:
+  #  - cron: '0 4 * * *' # run at 4 AM UTC, 12PM EST.
```

Comment:
Did you mean to leave these comments?

Reply:
Yes, I'm going to switch back to the schedule as soon as I'm finished testing. I am tracking a clean-up list before merging in the [PR Description](https://github.com/DataBiosphere/jade-data-repo/pull/637#issue-494321344).